### ISSUE TITLE
loss doesn't change each iter using softmax-fm?

### ISSUE THREAD

**Issue by tcsere07** [qqgeogor/fm_pypy#4]
loss doesn't change each iter using softmax-fm?
The training data is from two-sigma challenge.
